canonical-web-and-design/ubuntu.com
# fix Ubuntu Advantage's legal page links

## Done

- Fix the broken links in `/legal/ubuntu-advantage-service-terms`
- Change a deprecated link `/mssd` as mentioned in the [copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/ubuntu-advantage-service-terms or the [demo link](https://ubuntu-com-10995.demos.haus/legal/ubuntu-advantage-service-terms)
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure that the 4 links in the page are working
- Please resolve the discussion regarding the deprecated link in the [copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit)

## Issue / Card

Fixes #10994 


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
